src/cf-forms/src/molecules/form-fields.less

```diff
@@ -11,15 +12,19 @@
     &__checkbox,
     &__radio {
         .a-label {
+            display: table-row;
```

Comment:
Hrm, not working quite as expected, but I think for IE9 I'm not going to worry about it.

![screen shot 2017-01-05 at 6 49 39 pm](https://cloud.githubusercontent.com/assets/704760/21702157/e5ee4fee-d377-11e6-8181-2edc2739e8d6.png)

Also, the color is off in IE9. Same thing for the footer color.